modules/clients/src/test/java/org/apache/ignite/jdbc/JdbcAuthorizationTest.java

```diff
@@ -143,7 +144,11 @@ public void testInsert() throws Exception {
         assertAuthorizationFailed(sql, CACHE_READ_USER);
         assertAuthorizationFailed(sql, CACHE_REMOVE_USER);
 
+        int cnt = ignite(0).cache(DEFAULT_CACHE_NAME).size();
+
         execute(sql, CACHE_PUT_USER);
+
+        assertTrue(ignite(0).cache(DEFAULT_CACHE_NAME).size() > cnt);
```

Comment:
Can we verify that the cache size has only increased by one?

Reply:
Done.